flutter-studio/src/io/flutter/profiler/FlutterAllMemoryData.java

```diff
@@ -109,6 +119,15 @@ public void handleIsolatesInfo(VM vm, List<HeapMonitor.IsolateObject> isolates)
       public void handleGCEvent(IsolateRef iIsolateRef,
                                 HeapMonitor.HeapSpace newHeapSpace,
                                 HeapMonitor.HeapSpace oldHeapSpace) {
+        // TODO(terry): This event isn't being called?
```

Comment:
is this comment obsolete? Looks like GC events are showing up.

Reply:
They're showing up but as a result of the explicit manual GC being pushed the handleGCEvent is not being fired.  So I'd prefer to handle the event here once this works (VMService issue).